bats/creds.bats

```diff
@@ -19,17 +19,22 @@ teardown() {
     dolt creds new
     run dolt creds ls
     [ "$status" -eq 0 ]
-    declare -p lines
     [ "${#lines[@]}" -eq 1 ]
-    [[ "${lines[0]}" =~ (^\ \ ) ]] || false
+    [[ "${lines[0]}" =~ (^\*\ ) ]] || false
+    cred=`echo "${lines[0]}" | awk '{print $2}'`
+    dolt creds new
+    run dolt creds ls
+    [ "$status" -eq 0 ]
+    [ "${#lines[@]}" -eq 2 ]
+    # Initially chosen credentials is still the chosen one.
+    [[ "`echo "$output"|grep "$cred"`" =~ (^\*\ ) ]] || false
```

Comment:
Is this sufficient to exercise the new logic? Shouldn't we check the global config for the new key?

Reply:
The assert on line 23 is the thing that would have failed previously...there would have been no `*` at the start of the line for the newly created credential, because it wasn't selected. This assert is making certain that `dolt creds new` after one cred is selected doesn't change the selected one...it's sufficient for that AFAICT...it looks for the line in `dolt ls` that has that cred and makes sure it still starts with `*`. I don't think we should go looking at implementation details of the config storage, instead of testing through the dolt interfaces.